packages/react-composites/src/composites/CallComposite/components/CallArrangement.tsx

```diff
@@ -154,6 +156,8 @@ export const CallArrangement = (props: CallArrangementProps): JSX.Element => {
     };
   }
 
+  const captionsBannerProps = usePropsFor(_CaptionsBanner);
```

Comment:
Let's move all hook calls earlier to the top of the component closure

Reply:
fixed